Browser Support topic

Mulesoft-Mysti opened the pull request:
This one topic will replace all other content in docs.mulesoft.com related to browser support.

Commit:
adding browser support doc

Commit:
new browser topic

Commit:
revisions

Commit:
more fixes

Mulesoft-Mysti commented on modules/ROOT/nav.adoc:
```diff
@@ -1,11 +1,12 @@
 .xref:index.adoc[Anypoint Platform]
-* xref:index.adoc[MuleSoft Documentation]
+* xref:index.adoc[Documentation]
```
Just making the left nav simpler. The actual file topic is still MuleSoft Documentation, though it probably should be Anypoint Platform Documentation for accuracy--I think people search MuleSoft more than AP.

kbostrom replied:
Would you be willing to craft an entry for our Style Reference about this?
"Anypoint Platform" vs. "MuleSoft"?

Mulesoft-Mysti replied:
Absolutely. will follow up in slack.

Commit:
final tweaks to first draft

Mulesoft-Mysti commented on modules/ROOT/pages/browser-support.adoc:
```diff
@@ -0,0 +1,26 @@
+include::_attributes.adoc[]
+endif::[]
+
+// page-aliases: 
```
This line is commented out for now--other writers will be adding page aliases in the next sprint. So this line will change before being merged.

kbostrom commented on modules/ROOT/pages/browser-support.adoc:
```diff
@@ -0,0 +1,26 @@
+Use Anypoint Platform with the latest version of any supported browser:
+
+* Microsoft Edge with Chromium
+* Microsoft Edge wihtout Chromium (supported through December 31, 2020)
```
```suggestion
* Microsoft Edge without Chromium (supported through December 31, 2020)
```

kbostrom commented on modules/ROOT/pages/browser-support.adoc:
```diff
@@ -0,0 +1,26 @@
+* Mozilla Firefox
+* Safari
+
+The Anypoint Platform browser support follows the Salesforce Lightning Experience policy.
```
```suggestion
Anypoint Platform browser support follows the Salesforce Lightning Experience policy.
```
Will your audience be familiar with SFLE policy and what this means?

Mulesoft-Mysti replied:
A significant portion will not. Link is in the See Also.

kbostrom commented on modules/ROOT/pages/browser-support.adoc:
```diff
@@ -0,0 +1,26 @@
+== Unsupported Browsers
+
+Internet Explorer 11 is not supported by MuleSoft.
+While the browser may work with Anypoint Platform, you may experience issues.
```
```suggestion
While the browser might work with Anypoint Platform, you might experience issues.
```
Such as?

kbostrom commented on modules/ROOT/pages/browser-support.adoc:
```diff
@@ -0,0 +1,26 @@
+
+== See Also
+
+https://help.salesforce.com/articleView?id=getstart_browsers_sfx.htm&type=5[Supported Browsers for Lightning Experience^]
```
What's the little ^ doohickey for? I don't recall seeing anything like this before....

Mulesoft-Mysti replied:
It's for launching a new window instead of opening in the same window. We haven't gotten all this confirmed or documented yet in style guide, it's in process.

kbostrom reviewed (approved):
Edited

Mulesoft-Mysti committed:
Update modules/ROOT/pages/browser-support.adoc

Co-Authored-By: Kim Bostrom <18537445+kbostrom@users.noreply.github.com>

Commit:
fixing modals

Commit:
revised per Support

wwarner-salesforce committed:
add page alias for ACM browser support page

wwarner-salesforce committed:
formatting

fix page alias formatting to match example in https://wiki.corp.mulesoft.com/display/MTDT/Remove+or+Rename+Existing+Files
